### PR TITLE
Fix minor typos

### PR DESCRIPTION
**Pull request by az0**
### Description

This change is required to solve minor typographical errors. These changes do not affect any code.

### Related Issues

Not applicable

### Checklist:

- [X] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have added tests.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: notebooks/03_evaluate/evaluation.ipynb [hunk 1]
```diff
@@ -121,7 +121,7 @@
    "source": [
```
Comment (gramhagen) [inline notebooks/03_evaluate/evaluation.ipynb]: <p>I think there's still a type in the first sentence here, maybe a better version would be:</p><p>"For comparison, a similar process is used with a threshold value of 3 to create a more balanced dataset."</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/743/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

**Comment by gramhagen** [comment]
Thanks for the updates, one thing to note is we push PRs to staging instead of master, so I've changed the base used here. In this case I think it's okay because the differences are small, but normally I would suggest working off staging and pushing PRs from branches based on changes to the staging branch.

File: notebooks/03_evaluate/evaluation.ipynb (hunk 1 above)
Comment (az0) [reply notebooks/03_evaluate/evaluation.ipynb]: Great suggestion. I used it.

**Comment by az0** [comment]
I will try to be more careful about the branch. Thanks for the kind advice

**Comment by az0** [comment]
Replaced by https://github.com/Microsoft/Recommenders/pull/744

**Pull request closed without merging**